src/main/java/racing/Players.java

```diff
@@ -0,0 +1,44 @@
+package racing;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Players {
+    private List<Car> cars;
+
+    public Players(List<Car> cars) {
+        this.cars = cars;
+    }
+
+    public Players(String carNames) {
+        this.cars = insertCars(splitCarNames(carNames));
+    }
+
+    public String[] splitCarNames(String carNames) {
+        return carNames.split(",");
+    }
+
+    private List<Car> insertCars(String[] carNames) {
+        List<Car> temporaryCars = new ArrayList<>();
+
+        for (String carName : carNames) {
+            temporaryCars.add(new Car(carName));
+        }
+
+        return temporaryCars;
+    }
+
+    public List<String> getCarNames() {
+        List<String> carNames = new ArrayList<>();
+
+        for (Car car : cars) {
+            carNames.add(car.getName());
+        }
+
+        return carNames;
```

Comment:
```suggestion
       return Arrays.asList(carNames);
```
정적 클래스 `java.util.Arrays.ArrayList`로 리턴해 원소 추가/삭제를 방지해보는 건 어떨까요?

Reply:
List<Car>인 cars에서 Car의 String만을 뽑는 작업을 필요로 하여 Arrays.asList()만으로는 문제가 있습니다.